tests/test_chromatic.py

```diff
@@ -1970,15 +1970,12 @@ def __repr__(self):
     # check error messages from inconsistent pixel scales and image dimensions
     incorrect_ims = PSF.ims.copy()
     incorrect_ims[0].scale += 0.01
-    with np.testing.assert_raises(NotImplementedError):
-        galsim.InterpolatedChromaticObject.from_images(incorrect_ims, PSF.waves)
+    assert_raises(galsim.GalSimNotImplementedError, galsim.InterpolatedChromaticObject.from_images, incorrect_ims, PSF.waves)
     incorrect_ims[0].scale -= 0.01
     smaller_image = incorrect_ims[0].array[1:]
-    incorrect_img = galsim.Image(smaller_image, scale=scale)
+    incorrect_img = galsim.Image(smaller_image, scale=incorrect_ims[0].scale)
     incorrect_ims[0] = incorrect_img
-    with np.testing.assert_raises(NotImplementedError):
-        galsim.InterpolatedChromaticObject.from_images(incorrect_ims, PSF.waves)
-
+    assert_raises(galsim.GalSimNotImplementedError, galsim.InterpolatedChromaticObject.from_images, incorrect_ims, PSF.waves)
```

Comment:
FWIW, our assert_raises is identical to np.testing.assert_raises, so you could have left it as the context version if you prefer that for clarity.  But this is fine too.